### PR TITLE
Allowed supportedOperation on SupportedProperty.

### PR DESCRIPTION
**Pull request by alien-mcl**
## Summary

Allowing _supportedOperation_ predicate on _SupportedProperty_ instances.
It also marks several terms as *stable*.

## More details

This PR fixes issue #196. In general, there were doubts on how to (if possible) attach a supported operation to the property. There were few examples that did this by using _supportedOperation_ on the property itself, but this was incorrect neither from the raw range/domain point of view defined within the RDF spec nor semantic understanding of such approach.

This PR enables _supportedOperation_ to be formally attachable to instances of _SupportedProperty_. This way this intermediate structure can hold any API specific details preventing from leaking it to externally defined properties.

### CODE REVIEW

File: spec/latest/core/core.jsonld [hunk 1]
```diff
@@ -98,23 +98,23 @@
       "subClassOf": "rdfs:Resource",
       "label": "Hydra Resource",
       "comment": "The class of dereferenceable resources.",
-      "vs:term_status": "testing"
+      "vs:term_status": "stable"
```
Comment (tpluscode) [inline spec/latest/core/core.jsonld]: All of those changes appear unrelated to the matter at hand IMO

File: spec/latest/core/index.html [hunk 2]
```diff
@@ -601,6 +601,14 @@ <h3>Documenting a Web API</h3>
       for the operation invocation.
     </pre>
 
+    <p>To wrap up everything altogether, it is also possible to attach atomic
+      operations supported by, well, supported property itself. This might
+      come in handy for scenarios, when resource can be partially modified.
```
Comment (tpluscode) [inline spec/latest/core/index.html]: This is a little unclear without an example TBH. 

I used this feature for example on a "typed" collection link.

```turtle
# in data
<> a api:Entrypoint ; api:dataSheetCollection <data-sheets> .

# in API Documentation
api:Entrypoint
  hydra:supportedProperty [
    hydra:property api:dataSheetCollection ;
  ] .

api:dataSheetCollection
  a                        hydra:Link ;
  rdfs:range               <api#DataSheetCollection> ;
  hydra:supportedOperation _:PostToCollection .
```

This way my client dereferences `<>` (entrypoint) and discovers that it can POST to `<data-sheets>`.

----

So now that would change like so: 

```diff
api:Entrypoint
  hydra:supportedProperty [
    hydra:property api:dataSheetCollection ;
+   hydra:supportedOperation _:PostToCollection ;
  ] .

-api:dataSheetCollection
-  a                        hydra:Link ;
-  rdfs:range               <api#DataSheetCollection> ;
-  hydra:supportedOperation _:PostToCollection .
```

File: spec/latest/core/core.jsonld (hunk 1 above)
Comment (alien-mcl) [reply spec/latest/core/core.jsonld]: Ok. I didn't want to create a separate PR that is unrelated to any issue, but let it be that way. I'll update it ASAP.

File: spec/latest/core/index.html (hunk 2 above)
Comment (alien-mcl) [reply spec/latest/core/index.html]: Good point. I'll make required amendments to over that scenario.
In general, sole purpose was to make examples like `<Me> schema:knows <You>` obsolete as `schema:knows` is not `hydra:Link`. But your example clearly states that `api:dataSheetCollection` is a `hydra:Link`, thus it can have a `hydra:supportedProperty` attached.

Comment (alien-mcl) [reply spec/latest/core/index.html]: Done. I was wondering whether providing both examples could be useful.
